core/approval/service.go

```diff
@@ -126,6 +128,17 @@ func (s *Service) AdvanceApproval(ctx context.Context, appeal *domain.Appeal) er
 	return nil
 }
 
+func (s *Service) AddApprover(_ context.Context, approvalID, email string) error {
+	return s.repo.AddApprover(&domain.Approver{
+		ApprovalID: approvalID,
+		Email:      email,
+	})
+}
+
+func (s *Service) DeleteApprover(_ context.Context, approvalID, email string) error {
```

Comment:
why don't propagate context to DB?
in DB we use `db.WithContext(ctx)` as well

Reply:
created an issue for this https://github.com/odpf/guardian/issues/240